packages/twenty-ui/src/theme/constants/BackgroundDark.ts

```diff
@@ -24,6 +24,7 @@ export const BACKGROUND_DARK = {
     danger: RGBA(COLOR.red, 0.08),
   },
   overlay: RGBA(GRAY_SCALE.gray80, 0.8),
+  overlay2: RGBA(GRAY_SCALE.gray80, 0.4),
```

Comment:
overlayPrimary, overlaySecondary?

Reply:
yep, will make the change
This was just lazy on my part